bots/pr/src/test/java/org/openjdk/skara/bots/pr/CSRBotTests.java

```diff
@@ -20,60 +20,76 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-package org.openjdk.skara.bots.csr;
+package org.openjdk.skara.bots.pr;
 
-import org.openjdk.skara.bots.common.BotUtils;
-import org.openjdk.skara.bots.common.SolvesTracker;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
 import org.openjdk.skara.forge.PullRequestUtils;
-import org.openjdk.skara.issuetracker.Link;
 import org.openjdk.skara.issuetracker.Issue;
-import org.openjdk.skara.test.*;
+import org.openjdk.skara.issuetracker.Link;
 import org.openjdk.skara.json.JSON;
-
-import org.junit.jupiter.api.*;
+import org.openjdk.skara.test.CheckableRepository;
+import org.openjdk.skara.test.HostCredentials;
+import org.openjdk.skara.test.TemporaryDirectory;
+import org.openjdk.skara.test.TestBotRunner;
 
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
-import java.util.*;
+import java.util.List;
+import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.*;
-import static org.openjdk.skara.bots.common.PullRequestConstants.*;
 
 class CSRBotTests {
     @Test
     void removeLabelForApprovedCSR(TestInfo testInfo) throws IOException {
         try (var credentials = new HostCredentials(testInfo);
              var tempFolder = new TemporaryDirectory()) {
-            var repo = credentials.getHostedRepository();
-            var issues = credentials.getIssueProject();
-            var issue = issues.createIssue("This is an issue", List.of(), Map.of());
+            var author = credentials.getHostedRepository();
+            var reviewer = credentials.getHostedRepository();
+            var bot = credentials.getHostedRepository();
+            var issueProject = credentials.getIssueProject();
+            var issue = issueProject.createIssue("This is an issue", List.of(), Map.of());
+            issue.setProperty("issuetype", JSON.of("Bug"));
 
-            var csr = issues.createIssue("This is an approved CSR", List.of(), Map.of());
-            csr.setState(Issue.State.CLOSED);
-            csr.setProperty("resolution", JSON.object().put("name", "Approved"));
-            issue.addLink(Link.create(csr, "csr for").build());
+            var censusBuilder = credentials.getCensusBuilder()
+                    .addReviewer(reviewer.forge().currentUser().id())
+                    .addCommitter(author.forge().currentUser().id());
+            var prBot = PullRequestBot.newBuilder().repo(bot).issueProject(issueProject).censusRepo(censusBuilder.build()).enableCsr(true).build();
+            var csrIssueBot = new CSRIssueBot(issueProject, List.of(author), Map.of(bot.name(), prBot));
+
+            // Run issue bot once to initialize lastUpdatedAt
+            TestBotRunner.runPeriodicItems(csrIssueBot);
 
-            var bot = new CSRPullRequestBot(repo, issues);
+            var csr = issueProject.createIssue("This is an approved CSR", List.of(), Map.of());
```

Comment:
The title is misleading. The CSR isn't approved until the end of this test.

Reply:
Will fix it.